app/src/main/res/layout/are_you_sure_popup.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Note for the future (after this pull request): Android has a built-in `AlertDialog` class that takes care of modal dialogs like this.